configs/www/init.sls

```diff
@@ -212,9 +212,17 @@ get composer:
 
 # create a named vendor directory that won't clobber composer use in a project-specific repo
 # let the project overlay use the default "vendor" dir
+/var/www/html/api/vendor-lebp-stack-create:
+  file.directory:
+    - makedirs: true
+    - require:
+      - file: /var/www/html/api
+
 /var/www/html/api/vendor-lebp-stack:
   file.recurse:
     - source: salt://www/files/vendor-lebp-stack
+    - unless: test -d /var/www/html/api/vendor-lebp-stack
```

Comment:
This doesn't do what we would want, I don't think. This would prevent the sources in www/files/vendor-lebp-stack from being placed once the directory exists. Which would mean that it would always fail the same way the first time, and then would never push out any changed files once the directory had been created.

Reply:
Yea I will probably remove this one I thought it was a different outcome